src/components/Quotes/Quotes.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState, useEffect } from 'react';
+import './Quotes.css';
+
+const Quotes = () => {
+  const [quote, setQuote] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [haveError, setHaveError] = useState(false);
+
+  useEffect(() => {
+    const delay = 3000;
+
+    const fetchData = () => fetch('https://api.api-ninjas.com/v1/quotes?category=computers&limit=2', {
```

Comment:
Great work so far @hajnaloltyan  👍🏼  , I would just recommand using async wait key words when calling a process that takes time , if loading takes more than time you set this will cause a bag when trying to setQuotes to undefined 
`const fetchData =async () => {
      setIsLoading(true)
      try {
        const res = await fetch(
         'https://api.api-ninjas.com/v1/quotes?category=computers&limit=2',
          { .....
`